frigate/video.py

```diff
@@ -491,6 +494,8 @@ def process_frames(
             logger.info(f"{camera_name}: frame {frame_time} is not in memory store.")
             continue
 
+        motion_detector.improve_contrast = improve_contrast_enabled.value
```

Comment:
Can't you just pass `improve_contrast_enabled` to the constructor for the `MotionDetector` class? Then there is no need to set this on each loop.

Reply:
I don't think I can because after the class is instantiated, the value can change (similar to the way ``detection_enabled`` does also in ``process_frames``) after process creation. Right? Unless there's something I don't understand about Python classes, which could totally be the case.

Reply:
Well I just tried it and I guess you're right. Apologies for my lack of Python knowledge. I'll get this changed shortly.

Reply:
Alright, I think it's good now. Let me know how it looks. Thanks Blake!